packages/cli-kit/src/session/device-authorization.ts

```diff
@@ -14,6 +16,10 @@ export interface DeviceAuthorizationResponse {
   interval?: number
 }
 
+const DeviceAuthError = () => {
+  return new Abort('Failed to start authorization process')
```

Comment:
This is not very actionable. Should we make it a `Bug` to have some metrics on Bugsnag around how often and when it happens?

Reply:
Will do, there is nothing for the user to do... just try again.